earth_enterprise/src/fusion/autoingest/sysman/SConscript

```diff
@@ -161,7 +161,7 @@ env.test('LeafAssetVersion_unittest',
     LIBS=['gegdal', 'geraster', 'gesearchtabs', 'genet', 'gepublish', 'gefconfigutil',
       'geindex_r', 'geutil', 'gecommon', 'gexml', 'gemiscconfig', 'qt-mt', 'xerces-c', 'geos',
       'gtest', 'gomp'])
-
+'''
```

Comment:
This test needs to be added back in. Hopefully it will be easy to do at this point.